backend/apps/chat/consumers.py

```
@@ -125,47 +185,60 @@ def save_message(self, user, room_id, content, parent_id=None):
+
         if hasattr(self, "group_name"):
             if is_gone:
                 await self.channel_layer.group_send(
                     self.group_name,
                     {
                         "type": "presence_left",
+
+                        "username": username,
+                        "user_id": user_id,
+
                         "username": self.user.username,
```

Ruff (0.15.21)

[error] Dictionary key literal `"username"` repeated

(F601)